packages/common/src/hooks/useSortedData/useSortedData.ts

```diff
@@ -41,16 +41,14 @@ export const useSortedData = <T extends Record<string, unknown>>(
   const { sortBy, onChangeSortBy } = useSortBy(initialSortBy);
   const [sortedData, setSortedData] = useState(data);
 
-  const wrapped = (newSortRule: SortRule<T>) => {
-    const newSortBy = onChangeSortBy(newSortRule);
-    const sorter = getDataSorter(newSortBy.key, !!newSortBy.isDesc);
-    setSortedData(data.sort(sorter));
-  };
-
   useEffect(() => {
     const sorter = getDataSorter(sortBy.key, !!sortBy.isDesc);
     setSortedData(data.sort(sorter));
-  }, []);
+  }, [sortBy]);
+
+  useEffect(() => {
+    setSortedData(data);
+  }, [data]);
 
-  return { sortedData, sortBy, onChangeSortBy: wrapped };
+  return { sortedData, sortBy, onChangeSortBy };
```

Comment:
....

Hm.

So when the `data` changed from a parent and flowed down, the `sortedData` was still the same in state unless a sort occurred. So changed values weren't showing in the UI.

So again, need to revisit sorting 🙄  but this is working ok for this PR..